.github/workflows/create-release-branch.yml

```diff
@@ -93,8 +93,10 @@ jobs:
       - name: Synchronize package version reported to telemetry
         run: node common/scripts/sync-telemetry-package-version
 
-      - name: Add changelog
-        run: node ./common/scripts/changelog/change.mjs -m 'Create release branch and update SDK versions' --type none
+      - name: Disabled - DO THIS MANUALLY! Add changelog
```

Comment:
Are the docs updated to include doing this manually now?

Reply:
There is an ADO task to fix this and add this step back

Reply:
@JoshuaLai will be working on this, not sure if it will be done before the next release

Reply:
OK, make sure to notify the next release driver about this extra manual step